docs/generalTips/calculatorStructureV2/conditionalAlternatives.rst

```diff
@@ -1,39 +1,123 @@
+In this paradigm, one item in the calculator editor is provided with many alternative options, each with an conditional expression attached.
+Upon any change in the calculator, these expressions are evaluated, and **the last option whose expression evaluates to ``true`` is selected**.
+This option is then used in the calculator, while the rest are ignored.
+
+It may happen that among all alternatives provided, no expression evaluates to ``true``.
+In this case, no option will be used.
+In addition, a conditionless option will be treated as if its condition always evaluates to ``true``.
 
 Many fields in the calculator editor support conditional alternatives.
 These fields include:
 
 - :ref:`Variable block labels <variableAttributesV2Label>`;
 - :ref:`Variable block default values <variableAttributesV2DefaultValue>`;
 - :ref:`Variable block unit switchers <variableAttributesV2UnitSwitcher>`;
-- :ref:`Text blocks <textBlocks>` and their chunk variants; and
+- :ref:`Text blocks <textBlocks>`' chunk variants; and
 - :ref:`Image block URLs and captions <imageBlocks>`.
 
+Example: text block chunks and variants
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One text block consists of multiple chunks, whose contents are stitched together in-line:
+
+.. figure:: img/conditionalAlternatives/chunksBasic.png
+    :alt: A text block with two chunks.
+    :width: 90%
+    :align: center
+
+؜
+
+One chunk can have multiple variants, and up to one variant per chunk may be selected.
+Below, both variants' conditions are not provided, so they are both treated as ``true``:
+
+.. figure:: img/conditionalAlternatives/chunksVariantsNoConditions.png
+    :alt: A text block with two chunks. The last chunk has two variants.
+    :width: 90%
+    :align: center
+
+؜
+
+When conditions are provided, they are evaluated top-to-bottom and the last option that evalautes to ``true`` is selected.
+
+.. figure:: img/conditionalAlternatives/chunksXPositive.png
+    :alt: A text block with two chunks. The last chunk's first variant is "active".
+    :width: 90%
+    :align: center
+
+؜
+
+.. figure:: img/conditionalAlternatives/chunksXNegative.png
+    :alt: A text block with two chunks. The last chunk's second variant is "active".
+    :width: 90%
+    :align: center
+
+؜
+
+It may happen that no expression evaluates to ``true`` and no variant is selected:
+
+.. figure:: img/conditionalAlternatives/chunksNone.png
+    :alt: A text block with two chunks. No variant from the last chunk's variants is "active".
+    :width: 90%
+    :align: center
+
+؜
+
+It may happen that multiple variants' expressions all evaluate to ``true``, in which case the last of these variants will be selected:
+
+.. figure:: img/conditionalAlternatives/chunksMultipleTrue.png
+    :alt: A text block with two chunks. Only the last chunk is "active", despite multiple expressions evaluating to true.
+    :width: 90%
+    :align: center
+
+؜
```

Comment:
It's worth mentioning here again that a case where none of the conditions should apply should go at the top. For example, a "neither" option (neither positive nor negative) with no condition. If it is last, it will always be the last alternative that evaluates to true. This is different from most programming patterns, where the else case is last.